rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/utils/EventCodecUtil.java

```diff
@@ -40,6 +40,7 @@
 import org.ow2.proactive.scheduler.common.job.UserIdentification;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.ow2.proactive.scheduler.common.job.UserIdentification'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)